client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -16,63 +18,68 @@ import Gridicon from 'gridicons';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
 
-const PluginsBrowserListElement = React.createClass( {
+const PREINSTALLED_PLUGINS = [ 'Jetpack by WordPress.com', 'Akismet', 'VaultPress' ];
 
-	getDefaultProps: function() {
-		return {
-			iconSize: 40
-		};
-	},
+class PluginsBrowserListElement extends Component {
+	static defaultProps = {
+		iconSize: 40,
+	};
+
+	getPluginLink() {
+		if ( this.props.plugin.link ) {
```

Comment:
Any objections to using `get( this.props, 'plugin.link' )` here? We could safely avoid the guarding conditional if so.

Just a drive-by review & suggestion, FFTI :)